src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -476,10 +476,8 @@ m2mtest_bank(xclDeviceHandle handle, boost::property_tree::ptree& _ptTest, int b
   }
 
   //bandwidth
-  size_t total = bo_size;
-  total *= 1000000; // convert us to s
-  total /= (1024 * 1024); //convert to MB
-  return static_cast<int>(total / timer_stop);
+  double total_Mb = static_cast<double>(bo_size) / static_cast<double>(1024 * 1024); //convert to MB
+  return static_cast<double>(total_Mb / timer_duration_sec);
```

Comment:
No need to new case the value to a double since all of the variables are doubles.